index.html

```diff
@@ -5466,70 +5443,72 @@ <h2 id="AudioWorkletProcessor">
               </p>
             </dd>
             <dt>
-              void process()
-            </dt>
-            <dd>
-              <p>
-                Processes the audio data from the input and produce the content
-                of output buffer by executing the user-supplied script code.
-                This method is called synchronously by the audio <a>rendering
-                thread</a> that executes in an isochronous fashion.
-              </p>
-              <dl class="parameters">
-                <dt>
-                  sequence&lt;sequence&lt;Float32Array&gt;&gt; inputs
-                </dt>
-                <dd>
-                  <code>inputs[n][m]</code> is a <code>Float32Array</code> of
-                  audio samples for the <code>m</code>th channel of
-                  <code>n</code>th input. While the number of inputs is fixed
-                  at the construction, the number of channels can be changed
-                  dynamically.
-                </dd>
-                <dt>
-                  sequence&lt;sequence&lt;Float32Array&gt;&gt; outputs
-                </dt>
-                <dd>
-                  <code>outputs[n][m]</code> is a Float32Array object
-                  containing the audio samples for <code>m</code>th channel of
-                  <code>n</code>th output.
-                </dd>
-                <dt>
-                  Object parameters
-                </dt>
-                <dd>
-                  Provides an association between the names of the
-                  <a><code>AudioParam</code></a>s specified by the
-                  <a><code>AudioParamDescriptor</code></a>s during construction
-                  and the automation values for the
-                  <a><code>AudioParam</code></a>. Thus,
-                  <code>parameters["name"]</code> returns a
-                  <code>Float32Array</code> of the automation values
-                  corresponding to the <a><code>AudioParam</code></a> named
-                  <code>"name"</code>.
-                </dd>
-              </dl>
-            </dd>
-            <dt>
               void postMessage()
             </dt>
             <dd>
               <p>
-                It may be called to send a message to the
+                It may be called to send a message to the associated
                 <a><code>AudioWorkletNode</code></a>.
               </p>
               <dl class="parameters">
                 <dt>
                   any message
                 </dt>
                 <dd></dd>
-                <dt>
-                  optional sequence&lt;Object&gt; transfer
-                </dt>
-                <dd></dd>
               </dl>
             </dd>
           </dl>
+          <section class="informative">
```

Comment:
Because this section is built around the example of process() method, which cannot be a part of IDL. The text is also not normative; it is more of an explanation on how to extend the AWP class.